src/cmds/scripts/pbs_init.d.in

```diff
@@ -294,7 +294,7 @@ start_pbs() {
 		homever=`cat ${PBS_HOME}/pbs_version`
 		[ "${qstatver}" != "${homever}" ] && needmakehome=3
 	fi
-	[ ${needmakehome} -eq 0 -a "${PBS_START_SERVER}" != "0" -a ! -d "${PBS_HOME}/datastore" ] && needmakehome=4
```

Comment:
yes because we are not sure which daemon will be started first by the user so in that case if PBS_Home is not there that it has to be created, only datastore will be created and initialized when server will be started by user.